src/recipe/queries/handlers/findRecipeById.handler.ts

```
@@ -12,12 +10,6 @@ export class FindRecipeByIdHandler
 
   async execute(query: FindRecipeByIdQuery) {
     const { recipeId } = query;
-    const recipeFromDb = await this.recipeDAO.findRecipeById(recipeId);
-
-    if (!recipeFromDb) {
-      throw new NotFoundException();
-    }
-
-    return recipeFromDb;
+    return await this.recipeDAO.findRecipeById(recipeId);
```

you should throw notFoundException when recipe is not found